### PR TITLE
Feature/consumer api

### PR DESCRIPTION
**Pull request by ElRaptorus**
## What did you change?

Update the test environment with the changes made during the consumer API implementation.

This is actually kind of a legacy PR, since all consumer api functions were moved to the `consumer_api_meta` repo, but I think this updated testing environment might come in handy here aswell.

## How can others test the changes?

For testing the current setup, run the `setup.sh` script.

For testing the integationtests, move to `_integration_tests` and run `npm test`.

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

**Comment by Paulomart** [comment]
@ElRaptorus There are merge conflicts.

**Comment by ElRaptorus** [comment]
@Paulomart solved.

File: _integration_tests/application/ioc_module.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const registerInContainer = (container) => {
+
+  // TODO: Add processes for use in the integrationtests
```
Comment (Paulomart) [inline _integration_tests/application/ioc_module.js]: You should create an issue for that

Comment (ElRaptorus) [reply _integration_tests/application/ioc_module.js]: @Paulomart There was already an issue for the integrationtests. 
I've added a line for this topic there:
https://github.com/process-engine/process_engine_meta/issues/6

**Pull request merged**